pages/_app.css.d.ts

```diff
@@ -0,0 +1 @@
+
```

Comment:
Probably just leave this even though it's blank since autogen `css.d.ts` will always create it.